analyticsdata/test/analyticsDataTest.php

```diff
@@ -50,4 +50,20 @@ public function testClientFromJsonCredentials()
             $this->assertStringContainsString('does-not-exist.json', $ex->getMessage());
         }
     }
+
+    public function testRunReportWithNamedDateRanges()
+    {
+        $propertyId = self::requireEnv('GA_TEST_PROPERTY_ID');
+        $output = $this->runFunctionSnippet('run_report_with_named_date_ranges', [$propertyId]);
+
+        $this->assertRegExp('/Report result/', $output);
```

Comment:
Nit, just make this "assertStringContainsString", since we're not really using regex
```suggestion
        $this->assertStringContainsString('Report result', $output);
```